Appraisals

```diff
@@ -1,4 +1,4 @@
-%w(3.2 4.0 4.1 4.2).each do |version|
+%w(3.2 4.2 5.0).each do |version|
```

Comment:
`3.2` gemfiles are missing from `gemfiles` part in `travis.yml`. Is this correct?